app/views/dreamers/show.html.erb

```diff
@@ -1,4 +1,9 @@
 <h1 class="center"><%= @dreamer.username%>'s Profile</h1>
+
+<% if current_dreamer == @dreamer %>
```

Comment:
Can this be an" authorized?" method when we figure out the helper methods situation?